src/StartServerPlugin.js

```diff
@@ -36,8 +36,26 @@ export default class StartServerPlugin {
     return parseInt(port)
   }
 
+  _getSignal() {
+    const { signal } = this.options;
+    // allow users to disable sending a signal by setting to `false`...
+    if (signal === false) {
+      return;
+    }
+    // allow user to manually set the signal if they'd like
+    if (typeof signal === 'string') {
```

Comment:
`process.kill` takes both strings and numbers, so I'd just do `return signal || 'SIGUSR2'`